jobs/integr8ly/pds-general.yaml

```diff
@@ -0,0 +1,300 @@
+---
+
+- job:
+    name: pds-general
+    project-type: pipeline
+    description: "General pipeline for PDS testing."
+    sandbox: false
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 56
```

Comment:
Does it have any particular reason? Do we want to keep 56 builds?

Reply:
I would like to use this job to estimate stability so that we have a better understanding of what can be potentially executed per PR. I thought that the 8-week period should be long enough to evaluate this. 8 * 7 = 56.

Reply:
Interesting:-) OK Fine